WebContent/spinSetup/WOH-spinSetup.css

```diff
@@ -21,7 +9,8 @@ body
 	overflow-y: scroll; 
 }
 
-#rcorners4 {
+#rcorners4 
```

Comment:
Rename generic CSS classes and try to use classes instead of ids whenever possible.